easybuild/easyconfigs/i/Intltool/Intltool-0.51.0-intel-2015b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'Intltool'
+version = '0.51.0'
+
+homepage = 'http://freedesktop.org/wiki/Software/intltool/'
+description = """intltool is a set of tools to centralize translation of
+ many different file formats using GNU gettext-compatible PO files."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls  = ['http://launchpad.net/intltool/trunk/%(version)s/+download/']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('XML-Parser', '2.41', '-Perl-5.20.3'),
```

Comment:
I'm not sure. My gut feeling says it's ok here to not let it trickle up (since this is an uncommon dependency).